src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -201,7 +201,7 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
               } else {
                 const { type, reason } = error.err.attributes;
                 if (type === 'parsing_exception') {
-                  error.message = `Couldn't parse Elasticsearch ES|QL query. You may need to add backticks to names containing special characters. Check your query and try again. Error: ${reason}`;
+                  error.message = `Couldn't parse Elasticsearch ES|QL query. Check your query and try again. Error: ${reason}`;
```

Comment:
Can we use `i18n.translate` for error messages?

Reply:
@jughosta they are not translated anywhere atm, I have it on my list to check what is going to affect but I dont want to do this on this PR and so close to FF.